### PR TITLE
Refactor code around TitleSequence

### PR DESCRIPTION
**Pull request by ZehMatt**
NOTE: Not worth reviewing, a lot of stuff is just there to keep it working, a lot of lines will vanish once its complete.

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
Does this tie in with #10180?

**Comment by ZehMatt** [comment]
> Does this tie in with #10180?

Nah, this is just a general cleanup. Trying to get rid of the odd stuff like manual memory management

File: src/openrct2-ui/windows/TitleEditor.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*****************************************************************************
+﻿/*****************************************************************************
```
Comment (AaronVanGeffen) [inline src/openrct2-ui/windows/TitleEditor.cpp:1]: BOM?

Comment (ZehMatt) [reply src/openrct2-ui/windows/TitleEditor.cpp:1]: I have no idea why clang format is messing with me like this.

**Comment by ZehMatt** [comment]
All done, looks a lot better

**Review by tupaschoal — COMMENTED** [review]
Seems like this is doing part (or all of) #12815. I haven't personally checked one against the other. Can you do that, please?

Maybe you want to cherry-pick some commits from there to here and close the PR

**Comment by ZehMatt** [comment]
> Seems like this is doing part (or all of) #12815. I haven't personally checked one against the other. Can you do that, please?
> 
> Maybe you want to cherry-pick some commits from there to here and close the PR

Shit I haven't even realized this PR exists. I've done pretty much all of the other PR and more like getting rid of the utf8 types and fixing all memory leaks in there.

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**